src/Symfony/Bridge/PhpUnit/composer.json

```diff
@@ -16,7 +16,9 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9"
+        "php": ">=5.3.3 EVEN ON LATEST SYMFONY VERSIONS TO ALLOW USING",
+        "php": "THIS BRIDGE WHEN TESTING LOWEST SYMFONY VERSIONS.",
```

Comment:
Composer itself uses the underscore to add something like a comment: https://github.com/composer/composer/blob/master/composer.json#L41 Should we do that to to avoid confusion when reading this?